src/types/index.d.ts

```diff
@@ -10,3 +10,18 @@ declare type MaterialProps = {
   date: Date
   variant?: 'default' | 'small'
 }
+
```

Comment:
Here is a step-by-step guide that will help you remember all of the essential steps of writing a code review.

**How to communicate?**

- Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
- Remember to explain why you request a change - it might be obvious for you but not for someone else.
- Remember to **be polite** and clear in your comments - your job here is to help another person improve in a friendly way. ALWAYS use the magic word "Please..."
- You can include emojis and gifs to make your code review more human-friendly 👽.